### PR TITLE
All around LIKE implemented

### PR DESCRIPTION
**Pull request by messias-des-coden**
#56

### CODE REVIEW

**Review by dennisshushack — APPROVED** [review]
Gset guett uss

File: src/main/java/ch/uzh/ifi/hase/soprafs21/entity/Match.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
+/*package ch.uzh.ifi.hase.soprafs21.entity;
```
Comment (joelruettimann) [inline src/main/java/ch/uzh/ifi/hase/soprafs21/entity/Match.java:1]: Match entity already exist. Please pull master first, and  merge into personal branch

Comment (messias-des-coden) [reply src/main/java/ch/uzh/ifi/hase/soprafs21/entity/Match.java:1]: aight

File: src/main/java/ch/uzh/ifi/hase/soprafs21/service/LikeService.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+    @Autowired
+    public LikeService(@Qualifier("likeRepository") LikeRepository likeRepository) {
+        this.likeRepository = likeRepository;
+    }
+    public List<Like> getAllLikes(){
+        return this.likeRepository.findAll();
+    }
+
+    public Like createLike(Like likeInput) {
```
Comment (joelruettimann) [inline src/main/java/ch/uzh/ifi/hase/soprafs21/service/LikeService.java:26]: if a like already exists between the two ids a match should be crated

Comment (messias-des-coden) [reply src/main/java/ch/uzh/ifi/hase/soprafs21/service/LikeService.java:26]: I'm working on that

**Pull request merged**